tutorials/abap-environment-business-partner/abap-environment-business-partner.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
-title: Create Business Partner in S/4HANA Cloud using ABAP Environment Data
-description: Create a business partner in an S/4HANA Cloud system based on a business user in SAP Cloud Platform ABAP Environment.
+title: Create business partner in S/4HANA Cloud using ABAP Environment data
+description: Create a business partner in an S/4HANA Cloud system based on a business user in SAP BTP ABAP Environment.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.